src/web/lib/ArticleRenderer.tsx

```diff
@@ -136,6 +137,17 @@ export const ArticleRenderer: React.FC<{
                     );
                 case 'model.dotcomrendering.pageElements.ChartAtomBlockElement':
                     return <ChartAtom id={element.id} html={element.html} />;
+                case 'model.dotcomrendering.pageElements.QuizAtomBlockElement':
+                    return (
+                        <div id={`quiz-atom-${i}`}>
```

Comment:
Can we change this to use Figure? That component will also set the id but in addition it manages role, and vertical spacing.

The quiz atom element will need `role` too. This should be available from CAPI

Reply:
Quiz pages doe not have a role, so are we ok to keep this?

Reply:
Figure also does things like adding the `figure` tag for sematics as well as margins which might be useful. It's also just nice to be consistent, I think.

I suspect if you merge in `main` (where Figure is now widely used) this will make more sense